corehq/apps/geospatial/urls.py

```diff
@@ -25,6 +26,7 @@
     url(r'^gps_capture/json/$', get_paginated_cases_or_users_without_gps,
         name='get_paginated_cases_or_users_without_gps'),
     url(r'^gps_capture/$', GPSCaptureView.as_view(), name=GPSCaptureView.urlname),
+    url(r'^get_gps_users/$', get_users_with_gps, name='get_users_with_gps'),
```

Comment:
This URI is a bit weird, because it contains the method HTTP method ("get_") in the URI. Something like `users/` might be better suited, since the URI is already scoped to the geospatial "domain" (so whatever we return is used specifically for geospatial work) and we're calling it with a GET.

Reply:
Also, this view isn't getting called atm right? We're just adding it?

Reply:
That is a good point, I'll rename this to `users/json` to make the endpoint a little clearer.

> Also, this view isn't getting called atm right? We're just adding it?

That is correct, the front-end where this view is used will be done in a separate PR.

Reply:
[59ca8f4](https://github.com/dimagi/commcare-hq/pull/33471/commits/59ca8f4712db3f12157f78166284cfd44feaf94c)